ROMFS/px4fmu_common/init.d/rc.interface

```diff
@@ -179,52 +403,60 @@ then
 				set FAILSAFE_AUX none
 			fi
 		else
+			echo "ERROR: Could not start: fmu mode_pwm"
 			echo "ERROR: Could not start: fmu mode_pwm" >> $LOG_FILE
-			tone_alarm ${TUNE_ERR}
+			tune_control play -t 2
 			set PWM_AUX_OUT none
 			set FAILSAFE_AUX none
 		fi
 
-		# Set min / max for aux out and rates
+		# Set min / max for aux out and rates.
 		if [ $PWM_AUX_OUT != none ]
 		then
 			#
-			# Set PWM_AUX output frequency
+			# Set PWM_AUX max PWM values.
 			#
-			if [ $PWM_AUX_RATE != none ]
+			if [ $PWM_AUX_MAX != none ]
 			then
-				pwm rate -c ${PWM_AUX_OUT} -r ${PWM_AUX_RATE} -d ${OUTPUT_AUX_DEV}
+				pwm max -c ${PWM_AUX_OUT} -p ${PWM_AUX_MAX} -d ${OUTPUT_AUX_DEV}
 			fi
 
+			#
+			# Set PWM_AUX min PWM values.
+			#
 			if [ $PWM_AUX_MIN != none ]
 			then
 				pwm min -c ${PWM_AUX_OUT} -p ${PWM_AUX_MIN} -d ${OUTPUT_AUX_DEV}
 			fi
-			if [ $PWM_AUX_MAX != none ]
+
+			#
+			# Set PWM_AUX output frequency.
+			#
+			if [ $PWM_AUX_RATE != none ]
 			then
-				pwm max -c ${PWM_AUX_OUT} -p ${PWM_AUX_MAX} -d ${OUTPUT_AUX_DEV}
+				pwm rate -c ${PWM_AUX_OUT} -r ${PWM_AUX_RATE} -d ${OUTPUT_AUX_DEV}
 			fi
 		fi
 
-		# Set disarmed values for aux out
-
-		# Transitional support until all configs
-		# are updated
+		#
+		# Set disarmed values for aux out.
+		# Transitional support until all configs are updated.
+		#
 		if [ $PWM_ACHDIS == none ]
 		then
 			set PWM_ACHDIS ${PWM_AUX_OUT}
 		fi
 
 		#
-		# Set disarmed, min and max PWM_AUX values
+		# Set disarmed, min and max PWM_AUX values.
 		#
-		if [ $PWM_AUX_DISARMED != none -a $PWM_ACHDIS != none ]
+		if [ $PWM_AUX_DISARMED != none ]
 		then
 			pwm disarmed -c ${PWM_ACHDIS} -p ${PWM_AUX_DISARMED} -d ${OUTPUT_AUX_DEV}
```

Comment:
I just tested it on an X2.1 which has an IO, and this is executed as:
```
pwm disarmed -c  -p p:PWM_AUX_DISARMED -d /dev/pwm_output1
```
So `PWM_ACHDIS` is not set.

Reply:
@bkueng, I've pushed up a commit that deprecates `PWM_ACHDIS` and I certainly feel it improves/simplifies the logic flow in the script and config files.  Could you give me feedback given your test hardware setup at any time it is convenient?  Thanks!

Reply:
The change looks good